lib/biran/config.rb

```diff
@@ -45,10 +45,11 @@ def use_capistrano
     end
 
     def files_to_generate
-      {
-        vhost: {extension: '.conf'},
-        database: {extension: '.yml'}
-      }
+      { vhost: {extension: '.conf'} }
+    end
+
+    def db_file_to_generate
+      { database: { extension: '.yml' } }
```

Comment:
I don't understand this change. Why separate out the db file part?

Reply:
Because if we are removing the rails dependency is no longer a default file to generate. This PR still generate it by default, but having it in the default list of files will not allow it to filter it out easily.

Reply:
There is no need to generate vhost by default either. I picked sensible defaults for the most of our apps. It gets overridden by the values in an initializer or `config/app_config.yml`. So not sure what you mean by not filtered out easily. We have used biran on many projects that don't need the database.yml file. 
If your concern is removing something that is rails centric from the default values, then I'm okay with simply removing the file from the default list. But I don't see the need for all the extra logic on what should be some simple generic defaults. Plus, this change should have been in its own PR as I don't see how it affects the central problem of the gem not loading in a non rails app.

Reply:
Ok, I understand your concern, but this fails without a Rails.root existing. It might be a different issue, but this will never work without fixing this so it is coupled

Reply:
Its not needed to solve the problem. Removing the Rails.root is needed. You only simply need to remove the database line from the files to generate method returning the default options and fix the default path using Rails.root to satisfy your concerns about being too rails like.

Reply:
well, if I leave this file in there, Biran stops working ... so it is not working